Makefile

```diff
@@ -105,6 +109,21 @@ parsertrace_g: http_parser_g.o contrib/parsertrace.c
 tags: http_parser.c http_parser.h test.c
 	ctags $^
 
+install: library
+	install -D $(SONAME) $(LIBDIR)/$(SONAME)
```

Comment:
Can you add `INSTALL ?= install` at the top and use that instead of a literal `install`?